src/test/java/com/company/elixr/springbootapplicationwithdocker/FileOperationServiceTest.java

```diff
@@ -0,0 +1,164 @@
+        Assertions.assertNotNull(found);
+        Assertions.assertEquals(HttpStatus.OK, found.getStatusCode());
+        Assertions.assertEquals(2, found.getBody().getClass().getDeclaredFields().length);
+        Assertions.assertEquals(Constants.SUCCESS, found.getBody().getSuccess());
+        Assertions.assertEquals(fileInfo.getUserName(), found.getBody().getData().getUserName());
+        Assertions.assertEquals(fileInfo.getFileName(), found.getBody().getData().getFileName());
+    }
+
+    @Test
+    public void test_getFileById_IdNotInUUIDFormat() {
+
+        try {
+            String id = "ebadf9b2ea844683";
+        Mockito.when(fileOperationRepository.findById(Mockito.any(UUID.class)))
+                .thenThrow(new BadRequestException(Constants.ERROR_BAD_REQUEST_INVALID_ID_FORMAT));
+            fileOperationService.findFileById(id);
+        } catch (Exception exception) {
+            Assertions.assertNotNull(exception);
+            Assertions.assertEquals((BadRequestException.class).getName(),
+                    exception.getClass().getName());
+            Assertions.assertEquals(Constants.ERROR_BAD_REQUEST_INVALID_ID_FORMAT,
+                    exception.getMessage());
+        }
+    }
+
+    @Test
+    public void test_getFileById_NoRecordFound() {
+
+        UUID id = UUID.fromString("ebadf9b2-ea84-4683-bb32-a5bb4952e265");
+        Mockito.when(fileOperationRepository.findById(Mockito.any(UUID.class)))
+                .thenThrow(new NotFoundException(Constants.ERROR_NOT_FOUND));
+        try {
+            fileOperationService
+                    .findFileById(String.valueOf(id));
+        } catch (Exception exception) {
+            Assertions.assertNotNull(exception);
+            Assertions.assertEquals((NotFoundException.class).getName(),
+                    exception.getClass().getName());
+            Assertions.assertEquals(Constants.ERROR_NOT_FOUND,
+                    exception.getMessage());
+        }
+    }
+
+    @Test
+    public void test_getFileByUsername() {
+
+        Mockito.when(fileOperationRepository.findByUserName(Mockito.anyString()))
+                .thenReturn(fileInfoList);
+        ResponseEntity<SuccessResponse> found = fileOperationService.findFileByUserName(fileInfo.getUserName());
+        Assertions.assertNotNull(found);
+        Assertions.assertEquals(HttpStatus.OK, found.getStatusCode());
+        Assertions.assertEquals(Constants.STATUS, found.getBody().getStatus());
+        Assertions.assertEquals(fileInfo.getUserName(), found.getBody().getUserName());
+        Assertions.assertEquals(fileInfoList.get(0).getFileName(), found.getBody().getFiles().get(0).getFileName());
+    }
+
+    @Test
+    public void test_getFileByUserName_NoRecordFound() {
+
+        Mockito.when(fileOperationRepository.findByUserName(Mockito.anyString()))
+                .thenThrow(new NotFoundException(Constants.ERROR_NOT_FOUND));
+        try {
+            fileOperationService
```

Comment:
With the current code, the test will pass if no exception is thrown either. You need to explicitly fail the test in case no exception is thrown.

Reply:
Added Assertions.fail() to explicitly fail the test in case no exception is thrown.